src/toolkits/coreml_export/neural_net_models_exporter.cpp

```diff
@@ -141,8 +141,9 @@ void set_image_feature_size_range(ImageFeatureType* image_feature,
 ImageFeatureType* set_image_feature(
     FeatureDescription* feature_desc, size_t image_width, size_t image_height,
     std::string description = "",
-    ImageFeatureType::ColorSpace image_type = ImageFeatureType::RGB) {
-  feature_desc->set_name("image");
+    ImageFeatureType::ColorSpace image_type = ImageFeatureType::RGB,
+    std::string input_name = "image") {
```

Comment:
Let's not encode specific strings as defaults here. The caller should specify every time. I think `input_name` should be earlier in the argument list, too. It's more primary than e.g. `image_type`